### ISSUE TITLE
Alt-Tabbing with the steam overlay open in Overwatch 2 makes the overlay lose focus

### ISSUE THREAD

**Issue by simifor** [ValveSoftware/steam-for-linux#10567]
#### Your system information

* Steam client version (build number or date): 
* Distribution: Arch Linux 
* Opted into Steam client beta?: Yes (also tried without, no change)
* Have you checked for system updates?: Yes
* Steam Logs: 
[steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/14470221/steam-logs.tar.gz)

* GPU: Nvidia/Intel/AMD AMD RX 6600

#### Please describe your issue in as much detail as possible:
This issue seems to only affect OW2, when the overlay is open before we alt-tab, the game will claim cursor focus and the only way to make use of the overlay again is to toggle it off and on again.

On windows, when you alt-tab witht he overlay open, you'll see the cursor change to the OW one, but within half a second it'll change back to the system one and let you interact with the overlay.

Basically, in both linux and windows, when you alt-tab into the game, OW seems to steal the cursor focus. In windows, the overlay gets the focus back after a short time, but in linux  you have to toggle the overlay off and on again.

#### Steps for reproducing this issue:

1. open the game
2. open the steam overlay
3. alt-tab into another window and then back
4. can't interact with elements in the overlay
